### PR TITLE
[upmeter] fix slowing response of probe list

### PR DESCRIPTION
**Pull request by shvgn**
## Description

Pick groups and probes from runtime for the initial response for UIs

## Why do we need it, and what problem does it solve?

- Fixes the slow response on web UI's
- Cleans code for clarity

## Changelog entries

```changes
section: upmeter
type: fix
summary: Fixed slow data loading in
```

### CODE REVIEW

File: modules/500-upmeter/images/upmeter/pkg/constant/constant.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package constant
+
+// TODO move to config
```
Comment (shvgn) [inline modules/500-upmeter/images/upmeter/pkg/constant/constant.go:3]: move User-Agent values to config

Comment (shvgn) [reply modules/500-upmeter/images/upmeter/pkg/constant/constant.go]: done

File: modules/500-upmeter/images/upmeter/pkg/server/api/public_status.go [hunk 2]
```diff
@@ -122,16 +119,16 @@ func getStatusSummary(dbCtx *dbcontext.DbContext, monitor *crd.DowntimeMonitor)
 			muteDowntimeTypes: muteTypes,
 		}
 
-		resp, err := getStatus(dbCtx, monitor, filter)
+		resp, err := getStatus(h.DbCtx, h.DowntimeMonitor, filter)
 		if err != nil {
 			log.Errorf("cannot calculate status for group %s: %v", group, err)
-			continue
+			return groupStatuses, StatusOutage, err
```
Comment (diafour) [inline modules/500-upmeter/images/upmeter/pkg/server/api/public_status.go:125]: Just a note: it returns partial groupStatuses, may be `nil` is better?

**Review by diafour — APPROVED** [review]

File: modules/500-upmeter/images/upmeter/pkg/server/api/public_status.go (hunk 2 above)
Comment (shvgn) [inline modules/500-upmeter/images/upmeter/pkg/server/api/public_status.go:125]:
```suggestion
			return nil, StatusOutage, err
```

File: modules/500-upmeter/images/upmeter/pkg/server/api/public_status.go [hunk 3]
```diff
@@ -122,16 +119,16 @@ func getStatusSummary(dbCtx *dbcontext.DbContext, monitor *crd.DowntimeMonitor)
...
 		}
 
 		summary, err := pickSummary(ref, resp.Statuses)
 		if err != nil {
 			log.Errorf("cannot parse status for group %s: %v", group, err)
-			continue
+			return groupStatuses, StatusOutage, err
```
Comment (shvgn) [inline modules/500-upmeter/images/upmeter/pkg/server/api/public_status.go:131]:
```suggestion
			return nil, StatusOutage, err
```

File: modules/500-upmeter/images/upmeter/pkg/server/api/public_status.go (hunk 2 above)
Comment (shvgn) [reply modules/500-upmeter/images/upmeter/pkg/server/api/public_status.go]: Returning nil

**Comment by shvgn** [comment]
Closes #314

**Review by nabokihms — APPROVED** [review]

**Pull request merged**